benchmark/utils/DriverUtils.swift

```diff
@@ -198,8 +193,23 @@ struct TestConfig {
       }
     }
 
-    if let _ = benchArgs.optionalArgsMap["--run-all"] {
-      onlyPrecommit = false
+    if let x = benchArgs.optionalArgsMap["--skip-tags"] {
+      // if the --skip-tags parameter is specified, we need to ignore the
+      // default and start from a clean slate.
+      skipTags = []
+
+      // We support specifying multiple tags by splitting on comma, i.e.:
+      //
+      //  --skip-tags=array,set
+      //
+      // FIXME: If we used Error instead of .fail, then we could have a cleaner
```

Comment:
I am fine if you want to use Error here. Really we just need a better argument parsing framework. I have one that is partially finished if you want to look at it. Or maybe once we have swiftpm we can just take one from a package.

Reply:
I copied that comment from a block above. Did not really pay much attention to what it says. But yeah, a general purpose arg parsing lib would be nice in many places.